doc/nrf/includes/vsc_build_and_run_nrf53.txt

```diff
@@ -0,0 +1,14 @@
+1. Open Visual Studio code.
+#. Click on the :guilabel:`nRF Connect for VS Code` icon in the left navigation bar and then click :guilabel:`Open`.
+#. Click on :guilabel:`Add an existing application`.
+#. In the prompt, navigate to the folder containing the sample you want to build, such as :file:`nrf/samples/bluetooth/peripheral_lbs`.
+   You should now see the selected application in the :guilabel:`Applications` window.
```

Comment:
```suggestion
   The selected application appears in the :guilabel:`Applications` window.
```
I'd use something like this. "Should" can sound like there's some doubt whether the application will or will not appear.